_includes/footer.html

```diff
@@ -1 +1 @@
-<footer><span></span></footer>
+<footer><span>mlpack.org</span></footer>
```

Comment:
Do you think maybe we should put the mlpack logo here also?  The `mlpack.org` seems a bit plain to me.  Maybe we want to right-justify it or something too?

Reply:
Checkout the footer of this page, would that be a good txt footer? https://scikit-learn.org/stable/install.html

© 200? - 2019, mlpack developers (BSD License).

Logo would be redundant I guess...

Reply:
Ah yeah, that could work too. mlpack's first line of code was written in 2007 also, just like scikit. :)